L50-cpp-sts-creds-api.md

```diff
@@ -0,0 +1,89 @@
+Title
+----
+* Author(s): Julien Boeuf
+* Approver: a11r
+* Status: Draft
+* Implemented in: core, cpp
+* Last updated: May 20, 2019
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Is there a thread for this yet? Can you start one?

Reply:
Just did:
https://groups.google.com/forum/#!topic/grpc-io/Bam9eLaI62Q
Thanks for the reminder!